pkgs/applications/office/beamerpresenter/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, mkDerivation, fetchFromGitHub, makeDesktopItem,
+  qmake, qtbase, poppler, qtmultimedia }:
+
+mkDerivation rec {
+  pname = "beamerpresenter";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "stiglers-eponym";
+    repo = "BeamerPresenter";
+    rev = "v${version}";
+    sha256 = "0j7wx3qqwhda33ig2464bi0j0a473y5p7ndy5f7f8x9cqdal1d01";
+  };
+
+  enableParallelBuilding = true;
```

Comment:
Can be removed, parallel building is used by default for `qmake`.

Reply:
Removed and amended.